src/tests/golden/testdata/embeddedrules/offBy1.php

```diff
@@ -1,5 +1,9 @@
 <?php
 
+/**
+ * @param  mixed[]  $xs
+ * @param  mixed[]  $tabs
```

Comment:
I'm not sure why 2 spaces are used so consistently. Copy/paste maybe?

Reply:
Looks like IntelliJ auto format.

Reply:
Done.